manifest.json

```diff
@@ -41,11 +41,14 @@
     "32": "toggleQuotes32.png"
   },
   "commands": {
-    "_execute_browser_action": {
+    "_execute_message_display_action": {
       "suggested_key": {
         "default": "Alt+Q"
       },
       "description": "Toggle all quotes"
     }
-  }
+  },
+  "permissions": [
+    "messagesRead"
+  ]
```

Comment:
Needed for the messageDisplay API (to be notified if a new message is being displayed somewhere)